Automattic/wp-calypso
# Split the Global Styles variations in two: default and premium

Related to https://github.com/Automattic/dotcom-forge/issues/2261, https://github.com/Automattic/dotcom-forge/issues/2260

## Proposed Changes

* Split the default style variation from the Premium style variations in theme showcase and design picker

## Testing Instructions



* Use the live link from the comments
* Go to `site-setup/designSetup?siteSlug=your-site` (where the your-site is free)
* Click on a free theme and notice that style variations UI is split in two
* Resize your browser resolution to mobile and make sure that the UI looks good.
* Click on a Premium and Woo theme and notice that the UI is not split in two
* Repeat the same step with a site with a premium subscription and notice that all free, premium and Woo themes have the style variations combined in a single list. 
* Go to /themes (logged in and logged out)
* Make sure that for free sites, the UI is split in two for free themes
* Make sure that for free sites, the Ui is combined in a single list for Premium and Woo themes
* Make sure that for Premium sites the UI is combined for Free, Premium and Woo themes
* Make sure that the UI looks good on both mobile and desktop resolutions

## Screenshots
### Theme showcase
**Free site, free theme**
![Screenshot 2023-05-26 at 13 29 49](https://github.com/Automattic/wp-calypso/assets/995955/f2ff3b53-a71a-4d5d-a82f-cd774508427f)
**Premium site, free theme**
![Screenshot 2023-05-26 at 13 32 20](https://github.com/Automattic/wp-calypso/assets/995955/0b009ca9-6895-4fda-8033-0394ea811fee)

**Free site, free theme (mobile)**
<img width="366" alt="Screenshot 2023-05-31 at 21 12 29" src="https://github.com/Automattic/wp-calypso/assets/995955/bbaa2579-d664-4059-b178-d662c7221306">



### Design Picker
**Free site, free theme**
![Screenshot 2023-05-26 at 13 35 55](https://github.com/Automattic/wp-calypso/assets/995955/6901d262-1edd-46b9-b52e-87a7b790ddbf)

**Free site, free theme (mobile)**

<img width="427" alt="Screenshot 2023-05-31 at 21 17 18" src="https://github.com/Automattic/wp-calypso/assets/995955/28fb5d76-ad57-427d-a58d-bfcf8db016dc">


## Pre-merge Checklist
